apps/spark/src/test/java/com/linkedin/openhouse/catalog/e2e/CatalogOperationTest.java

```diff
@@ -0,0 +1,24 @@
+package com.linkedin.openhouse.catalog.e2e;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.linkedin.openhouse.tablestest.OpenHouseSparkITest;
+import org.apache.spark.sql.SparkSession;
+import org.junit.jupiter.api.Test;
+
+public class CatalogOperationTest extends OpenHouseSparkITest {
+  @Test
+  public void testCasingWithCTAS() throws Exception {
```

Comment:
Haven't really looked into the implementation, but the test cannot verify if the casing issue is fixed. CTAS is never a problem and this test can pass without the fix. Remember the issue happens in the write path not read? Please try the following test to verify:
```
spark.sql("create table openhouse.d1.tT1 (name string)")
spark.sql("insert into openhouse.d1.tt1 values ('foo')")
```

Reply:
Adding e2e catalog tests in `apps` is a stretch to the `apps` module context. Should the catalogTests suite be added in `integrations` module?

Reply:
@rohitkum2506  good point. I am moving them into spark-runtime module.

Reply:
resolving this thread as both comments have been addressed.